MapboxCoreNavigation.podspec

```diff
@@ -42,7 +42,7 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxNavigationNative", "~> 3.1"
   s.dependency "MapboxDirections.swift", "~> 0.24.0"
-  s.dependency "MapboxMobileEvents", "~> 0.5"
+  s.dependency "MapboxMobileEvents", "~> 0.5.0"
```

Comment:
We should have a comment in the spec that explicitly calls out the need for `0.x.0` pinning schema.

Reply:
Yes, we should be doing this consistently to avoid external breakage